packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte

```diff
@@ -13,6 +13,34 @@ onMount(async () => {
 
 async function updateProxySettings() {
   await window.updateProxySettings(proxySettings);
+
+  // loop over all providers and container connections to see if there are any running engines
+  let runningProviders = false;
+  const providerInfos = await window.getProviderInfos();
+  providerInfos.forEach(async provider => {
```

Comment:
Could be rewritten more readable with filter, something like

`const runningProviders = providersInfo.filter(p => p.containerConnections.filter(c => c.status !== 'stopped).length > 0).length > 0`

Reply:
Fixed.